colorcoder/Color.java

```diff
@@ -0,0 +1,5 @@
+package colorcoder;
```

Comment:
Significance of IColor interface

Reply:
To retrieve the color name of matching index from minor and major enum color which shares common method getindex() in-order to access generic methods.